external_server/checkers/mqtt_session.py

```
@@ -42,7 +42,7 @@ def set_id(self, session_id: str) -> None:
         """Update the session ID."""
         if not session_id:
             raise ValueError("Session ID cannot be empty")
-        _logger.info(f"Updating session ID from '{self._id}' to '{session_id}'.", self._car_name)
+        _logger.info("Setting the MQTT session ID.", self._car_name)
```

Maybe consider logging the session id when it is being set during the connection sequence. The logs should not be spammed by long session IDs, since that  was a bug in module gateway. Also a check for session ID length could be implemented? Currently the length should always be 8, but that could change in the future. So maybe throw an error log if the session ID is larger than 100 (or some other reasonable constant value). This way, if the bug occurs again, we will be notified.

---

I added the session ID to the log. The maximum length of the session ID, after which error is logged (I would prefer NOT raisin an exception) should be configurable value present in the config. I suggest to skip adding this check for now.